core/app/models/spree/product.rb

```diff
@@ -184,13 +184,13 @@ def property(property_name)
       product_properties.find_by(property: prop).try(:value)
     end
 
-    def set_property(property_name, property_value)
+    def set_property(property_name, property_value, property_presentation = property_name)
```

Comment:
I could, but IMHO this way looks cleaner.

AFAIK, it will not cause warnings in any ruby version (I have tried it with 1.8.7, 2.1.5, 2.2.1 & jRuby 9k pre1).